cmd/local/install.go

```diff
@@ -124,6 +118,11 @@ func initVarDir() error {
 	return nil
 }
 
+func initDataVolumeDir() error {
+	dataPath := strings.Split(dataVolume, ":")[0]
+	return createDirIfNotExists(dataPath)
```

Comment:
As mentioned in the other comment
not because the function is named wrong, you are using it in an incorrect way
The directory is created using `sudo` and files under it get's permissions set onto them
This is not the intended use

just use `os.MkdirAll`

Reply:
change to os.MkdirAll